apps/docs/docs/ref/kotlin/installing.mdx

```diff
@@ -0,0 +1,267 @@
+---
+id: installing
+title: 'Installing & Initialization'
+slug: installing
+custom_edit_url: https://github.com/supabase/supabase/edit/master/web/spec/supabase.yml
+---
+
+
+### Add one or more modules to your project
+
+<RefSubLayout.EducationRow>
+    <RefSubLayout.Examples>
+        <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+            <p style={{ marginRight: '8px' }}>Newest version:</p>

```

Comment:
Nit: You could skip the "Newest version", I believe the two images should be sufficient